e2e/cypress/integration/plugins/upgrade_spec.js

```diff
@@ -25,13 +25,14 @@ import * as TIMEOUTS from '../../fixtures/timeouts';
 
 describe('Plugin remains enabled when upgraded', () => {
     const pluginIdDemo = 'com.mattermost.demo-plugin';
+    const demoPluginURL = 'https://github.com/mattermost/mattermost-plugin-demo/releases/download/v0.1.0/com.mattermost.demo-plugin-0.1.0.tar.g';
```

Comment:
@hanzei the 2nd test was failing to download the plugin even after your last merge. 

Seems there is an error in the URL the `z` is missing here in `com.mattermost.demo-plugin-0.1.0.tar.g`
Adding the `z` to the end fixed it for me.

Reply:
Ah, I missed that. Nice catch, Dylan!

Reply:
Good catch @DHaussermann :+1: